### PR TITLE
Added TraceID

### PR DESCRIPTION
**Pull request by andriikonstantinov**

### CODE REVIEW

**Review by zimnx — APPROVED** [review]

File: extension/goplugin/environment.go [hunk 1]
```diff
@@ -287,6 +298,7 @@ func (thisEnvironment *Environment) LoadExtensionsForPath(extensions []*schema.E
 }
 
 func (thisEnvironment *Environment) dispatchSchemaEvent(prioritizedSchemaHandlers PrioritizedSchemaHandlers, sch Schema, event string, context map[string]interface{}) error {
+	thisEnvironment.Logger().Debugf("Starting event: %s, schema: %s", event, sch.rawSchema.ID)
```
Comment (przemyslaw-dobrowolski-cl) [inline extension/goplugin/environment.go:301]: Why this log is a part of this commit?

File: extension/goplugin/environment.go [hunk 2]
```diff
@@ -167,11 +179,10 @@ func (thisEnvironment *Environment) Start() error {
...
 
 	// Bind
-	thisEnvironment.extCore = NewCore(thisEnvironment)
-	thisEnvironment.extLogger = NewLogger(thisEnvironment)
-	thisEnvironment.extSchemas = NewSchemas(thisEnvironment)
-	thisEnvironment.extSync = NewSync(thisEnvironment)
-	thisEnvironment.extDatabase = NewDB(thisEnvironment)
+	thisEnvironment.Bind()
+
+	//Generating TraceID
```
Comment (przemyslaw-dobrowolski-cl) [inline extension/goplugin/environment.go]: In all places, after comment slases there is one space, but not here

File: extension/goplugin/environment.go [hunk 3]
```diff
@@ -616,5 +620,10 @@ func (thisEnvironment *Environment) Clone() extension.Environment {
 
 		initFnRaw: thisEnvironment.initFnRaw,
 		initFns:   thisEnvironment.initFns,
+
+		traceID: uuid.NewV4().String(),
 	}
+	env.Bind()
```
Comment (przemyslaw-dobrowolski-cl) [inline extension/goplugin/environment.go]: Why bind is public? It is dangerous

**Review by przemyslaw-dobrowolski-cl — CHANGES_REQUESTED** [review]

File: extension/goplugin/environment.go (hunk 1 above)
Comment (p-kozlowski) [inline extension/goplugin/environment.go:301]: defer a log message at the end of this function

**Review by p-kozlowski — CHANGES_REQUESTED** [review]

File: extension/goplugin/environment.go (hunk 1 above)
Comment (p-kozlowski) [reply extension/goplugin/environment.go:301]: IMO it's good to have it

Comment (andriikonstantinov) [reply extension/goplugin/environment.go:301]: Done

**Review by p-kozlowski — APPROVED** [review]

**Comment by p-kozlowski** [comment]
@przemyslaw-dobrowolski-cl I'd like to merge this PR. are you satisfied with the changes?

**Comment by przemyslaw-dobrowolski-cl** [comment]
ok

**Review by przemyslaw-dobrowolski-cl — APPROVED** [review]
ok

**Pull request merged**